core/explorer/core/model.py

```diff
@@ -26,6 +26,13 @@ class OutputPathName(Enum):
     DEV = "dev"
 
 
+class OutputPath(BaseModel):
+    """A Nix outputPath"""
+
+    name: OutputPathName = Field(description="The output path's name")
```

Comment:
Could we change that to `str` to fix https://github.com/tweag/nixpkgs-graph-explorer/issues/67 ?

Reply:
Yep! Done here: 616db1f47cadbc228dd55f82cd6a72aa757abaaa